.stylish-haskell.yaml

```diff
@@ -198,7 +198,7 @@ steps:
 # Note: we tend to write code which fits into 80 characters, in some cases
 # 100 is acceptable. For imports we always permit 100 characters because it
 # decreases verbosity of diffs and makes merging easier.
-columns: 100
+columns: 80
```

Comment:
The comment should be revised if we go with 80.

Reply:
I *really* wonder about 80. I can see not wanting to go above about 120 (github side-by-side diffs obscure the far right of a line), but 80 just doesn't seem to make sense, especially with 4 space indent. I could possibly live with 80 columns if the indent was 2 spaces.

The current code does roughly adhere to 100 columns. It does not adhere to 80.

Reply:
Yeah, I second Erik here. For me 80 columns is a bit too short in terms of screen-estate, especially since we are using 4 spaces indentation 😁

Reply:
you shouldn't be nesting indents so much :wink: 

The current code should be adhering to 80 columns -- that's what's in the .editorconfig and the style guide. If we want to change that then we should change the relevant docs/editorconfig stuff.